lib/prompts/promptUtils.ts

```diff
@@ -1,24 +1,45 @@
-// @ts-nocheck
 const inquirer = require('inquirer');
 
-const promptUser = async promptConfig => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const promptUser = async (promptConfig: any) => {
   const prompt = inquirer.createPromptModule();
   return prompt(promptConfig);
 };
 
-const confirmPrompt = async (message, defaultAnswer = true) => {
+export const confirmPrompt = async (
```

Comment:
Nit: we should use the `function` keyword over `const` when writing most TS functions. Pretty sure that was in the official HubSpot TS guidelines, although I can no longer find it. But it looks a lot cleaner too.

Reply:
done in [cc915d2](https://github.com/HubSpot/hubspot-cli/pull/1222/commits/cc915d22819ba914fcdfb84f88f28984e110cb07)